testnet/stacks-node/src/tests/l1_observer_test.rs

```diff
@@ -218,6 +255,14 @@ fn l1_integration_test() {
     // Give the run loop time to start.
     thread::sleep(Duration::from_millis(2_000));
 
+    let burnchain = Burnchain::new(
+        &config.get_burn_db_path(),
+        &config.burnchain.chain,
+        &config.burnchain.mode,
+    )
+    .unwrap();
+    let sortition_db = SortitionDB::open(&burnchain.get_db_path(), true).unwrap();
```

Comment:
```suggestion
    let (sortition_db, burndb) = burnchain.open_db(true).unwrap();
```

And then you can delete line 319 as well.